sql/core/src/main/scala/org/apache/spark/sql/DataFrameNaFunctions.scala

```diff
@@ -196,6 +196,24 @@ final class DataFrameNaFunctions private[sql](df: DataFrame) {
   def fill(value: String, cols: Seq[String]): DataFrame = fillValue(value, cols)
 
   /**
+   * Returns a new `DataFrame` that replaces null values in boolean columns with `value`.
+   */
+  def fill(value: Boolean): DataFrame = fill(value, df.columns)
+
+  /**
+   * (Scala-specific) Returns a new `DataFrame` that replaces null or NaN values in specified
```

Comment:
Oh, right. I copied the defs and docs from double, as it shows. Will change, NaN booleans would be weird indeed